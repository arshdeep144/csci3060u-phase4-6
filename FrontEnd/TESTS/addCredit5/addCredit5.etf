10 admin02 AD 9998
06 user03 FS 1
00 admin02 AD 9998
