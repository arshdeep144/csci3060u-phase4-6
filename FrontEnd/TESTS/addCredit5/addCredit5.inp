login
admin02
addcredit
user03
1
logout