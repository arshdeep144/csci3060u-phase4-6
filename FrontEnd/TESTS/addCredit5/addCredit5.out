Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Welcome to the main menu. Please enter a command to continue: 
Please enter a username
hello world
hello world
Please enter how much credit you would like to add
Adding credits...
Subtracting credits...
New credits for Admin are: 9998 and new credits for user are: 2949
Welcome to the main menu. Please enter a command to continue: 
